### DISCUSSION TITLE
When will the Helm chart drop Airflow 1.x support (or any other older version?)

### DISCUSSION THREAD

**Discussion by topherinternational** [apache/airflow#45101] [Q&A] [answered]
Looked in the chart docs, couldn't find an answer so asking here - 

I just put up a PR (#45082) involving some settings that need to be supported two different ways in the Helm chart bc Airflow changed how those prefs are wired and the chart supports Airflow versions all the way down to 1.10.x.

So I got to thinking, 

1. Is there a plan/milestone/etc to drop support for Airflow 1.x, or any "legacy" version including 2.x versions? (where "x" = a long time before the current 
2. What do the Airflow honchos consider a major-version change in chart behavior? 
a. Does raising the minimum supported Airflow versions (or the supported versions of other things like gitsync) constitute a breaking change/major-version bump? 

Supporting backcompat for prefs isn't a huge challenge, but it does mean writing more code and tests, and requires effort to note/comment the code where it can be deleted once support is dropped for older versions.

**Comment by potiuk** [comment, marked as answer]
This is a very good point. 

We have not discussed it (@jedcunningham @ephraimbuddy @utkarsharma2 @eladkal - as you are release mangers, to your attention).  

But I personally it would be reasonable to make our chart follow similar approach as providers and have rolling support for 2.X.+ (maybe for Chart it would make sense to have 12 months support after 2.x.0 or 3.x.0 is released)

But that is a very important topic that requires a devlist conversation. 

@topherinternational -> can you please move that discussion to devlist (see https://airflow.apache.org/community/) ?

**Reply by topherinternational** [reply]
> @topherinternational -> can you please move that discussion to devlist (see https://airflow.apache.org/community/) ?

Will do. I have a question however - when I post to the list, will my email address be visible in full or obscured somehow? (That will influence which email account I use, and I don't want to try it and find out 😅 )
